server/src/test/java/org/elasticsearch/common/BackoffPolicyTests.java

```diff
@@ -77,6 +77,25 @@ public void testExponentialBackOff() {
         }
     }
 
+    public void testLinearBackoffWithLimit() {
```

Comment:
Maybe also a test without the limit?

Reply:
Added in ab10b44d423834e53e468362536879edbadda5b6